app/sidekiq/copay_notifications/new_statement_notification_job.rb

```diff
@@ -26,12 +26,20 @@ class NewStatementNotificationJob
       end
     end
 
+    sidekiq_retries_exhausted do |_msg, ex|
+      StatsD.increment("#{STATSD_KEY_PREFIX}.failure")
```

Comment:
**Suggestion:** `STATSD_KEY_PREFIX` is being used before it's defined on line 39. This block should be moved below it so it's more readable.